src/isconnected.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# This script checks if a Bluetooth device is connected at a specific address.
+# This script can be called in this format:
+# 	./is_connected.sh $BTADDR 
+
+address=$1
+
+if echo info $address | sudo bluetoothctl | grep -q 'Connected: yes'; then
```

Comment:
See if you can find a better way to do this. Piping the command to grep is slow and not a great solution. I'm sure there's a better way to do it without having to do string searching. Try using devices in bluetoothctl.

Reply:
devices in bluetoothctl only shows paired devices, making no distinction if they are available or even connected.

Reply:
There's other ways (outside of bluetoothctl) to find connected devices. I don't like this grepping method. It's not going to be horribly slow because there's only like 10 lines of output and regex is fast, but there's gotta be a better way.